v2/azure-arm.yaml

```
@@ -883,6 +883,12 @@ objectModelConfiguration:
       Service_Product:
         $exportAs: Product
         $supportedFrom: v2.4.0
+      Service_Products_Api:
+        $exportAs: ProductApi
+        $supportedFrom: v2.4.0
```

Assuming we target getting this into the upcoming release, this needs to be `v2.5.0`

```suggestion
        $supportedFrom: v2.5.0
```